src/assets/js/archive.js

```diff
@@ -122,66 +125,93 @@ function createTableCell(content, dataClass) {
 }
 
 function updateTableFailed(os) {
-  const tab = $(`#tab-os-${os}`);
-  tab.find('.loading').text('Failed to load releases. Refresh page to try again.');
+  const tab = document.getElementById(`tab-os-${os}`);
+  const loadingElements = tab.querySelectorAll('.loading');
+  loadingElements.forEach(function (element) {
+    element.textContent = 'Failed to load releases. Refresh page to try again.';
+  });
 }
 
 let macOSArm64ArchiveFilename = '';
 
 // Listen for the macOS arm64 download link to be clicked and update
 // the example unzip command with correct arm64 filename
-$('.download-latest-link-macos-arm64').click(function () {
-  // Update inlined filenames in <code> element text nodes with arm64 filename:
-  const fileNamePrefix = 'flutter_';
-  const code = $(`code:contains("${fileNamePrefix}")`);
-  const textNode = $(code).contents().filter(function () {
-    return this.nodeType === 3 && this.textContent.includes(fileNamePrefix);
+(() => {
+  document.querySelector('.download-latest-link-macos-arm64')
+    .addEventListener('click', function () {
+      // Update inlined filenames in <code> element text nodes with arm64 filename:
+      const filteredElements = filterCodeElements();
+
+      filteredElements.forEach(function (node) {
+        node.textContent = `unzip ~/Downloads/${macOSArm64ArchiveFilename}`;
```

Comment:
This seems to replace the entire code block, when we only want to replace the filename. Perhaps we can reuse some of the regex logic from below setting the filename the first time.

**Before clicking:**

<img width="567" alt="Code block with all text" src="https://github.com/flutter/website/assets/18372958/b500dc03-c382-47e7-853c-59fb70d33d74">

**After clicking (with this change):**

<img width="558" alt="Code block with all text replaced" src="https://github.com/flutter/website/assets/18372958/72788b3d-829e-4d7d-9e7c-875015ba2b8f">

**After clicking (expected):**

<img width="623" alt="Code block on live docs.flutter.dev" src="https://github.com/flutter/website/assets/18372958/2fd1c310-e4cc-423c-bcb6-8273c63a94ae">

Let me know if you need any help or pointers fixing this one.

Reply:
I found the reason for this and I will work on this tomorrow. Tnks